glean-core/rlb/tests/init_fails.rs

```diff
@@ -10,7 +10,7 @@
 
 mod common;
 
-use std::{thread, time::Duration};
+use std::{path::PathBuf, thread, time::Duration};
```

Comment:
With the below this probably can be removed again.

```suggestion
use std::{thread, time::Duration};
```